src/components/Login/LogInForm.jsx

```diff
@@ -0,0 +1,33 @@
+import { ToastContainer } from 'react-toastify';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+
+const LogInForm = ({ handleSignInSubmit, handleChange }) => (
+  <form onSubmit={handleSignInSubmit} className="modal-form">
+    <div>
+      <ToastContainer />
+    </div>
+    <label htmlFor="email">
```

Comment:
Hello @Philipotieno ,
You could use placeholders instead of labels. It would make the UI be more appealing.

Reply:
Thanks for this its done